docs/tutorial/04_integrate_sdcore_with_observability.md

```diff
@@ -4,10 +4,10 @@ We will integrate the 5G core network with the Canonical Observability Stack (CO
 
 ## Enable the `metallb` MicroK8s addon
 
-Enable the `metallb` addon with a range of 1 IP address:
+Enable the `metallb` addon with a range of several addresses:
 
 ```console
-sudo microk8s enable metallb 10.0.0.2-10.0.0.2
+sudo microk8s enable metallb 10.0.0.2-10.0.0.10
```

Comment:
As discussed in the daily, let's not allocate more IPs than we actually need.

Reply:
I must have missed this part of the discussion. I'm curious what the reasoning behind that decision is.

Reply:
Let me reverse the question - what would be the reasoning for allocating more IPs that we need?
I know we could allocate 15 IPs and then we wouldn't have to worry next time we add LB, but then again I think there is some value in showing that `that's the minimal amount of resources needed to run your 5G core`.

Reply:
Fair enough. I wish there was a way to detect the amount we need and dynamically set it.

I'll make this change once I finish up with some other changes (I'll need to re-test the tutorial, so it will take a little bit).

Reply:
Fixed in https://github.com/canonical/charmed-5g/pull/125/commits/414760b4c77987b491c0bc0aa6d88a4d85a80bb3